### PR TITLE
Fix disallowed html and duplicate titles

### PR DESCRIPTION
**Pull request by LJSpiller**

### CODE REVIEW

**Comment by LJSpiller** [comment]
The validation errors are being addressed in other PRs

File: dynamics-gp/installation/installing-on-subsequent-computers.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Client installation"
+title: "Installing Dynamics GP on subsequent computers"
```
Comment (v-andreaco) [inline dynamics-gp/installation/installing-on-subsequent-computers.md:2]:
```suggestion
title: "Installing Dynamics GP on subsequent computers"
```
Per MS Style, we should be using imperative for titles and headings. :)

Comment (LJSpiller) [reply dynamics-gp/installation/installing-on-subsequent-computers.md:2]: @v-andreaco - it looks this was already done. Thanks!

Comment (v-andreaco) [reply dynamics-gp/installation/installing-on-subsequent-computers.md:2]: @LJSpiller 
Do you mean in another PR, maybe? In most of the titles that were changed in this PR, the titles are using gerunds when they should be imperative. Have they been changed somewhere else?

**Comment by LJSpiller** [comment]
Hello @edupont04 and @v-andreaco,

This PR addresses disallowed html and duplicate titles. The alt text validation issues were being addressed in another PR by another team member. I will be working on other validation issues in a new PR.  It is ready for final review.

File: dynamics-gp/installation/installing-on-subsequent-computers.md (hunk 1 above)
Comment (LJSpiller) [reply dynamics-gp/installation/installing-on-subsequent-computers.md:2]: @v-andreaco - Sorry, I thought you had made an edit. The remaining titles that were not duplicates used gerunds. I matched syntax for the edited titles. But yes, they should be imperatives. I can change all of the titles in another PR.

Comment (edupont04) [reply dynamics-gp/installation/installing-on-subsequent-computers.md:2]: Specifically for this docset, the general style guide does not fully apply. The content is converted from older whitepapers and manuals, written in a different time and age. Fix gerunds to imperative if you like/can automate, but it shouldn't be a hard requirement.

Comment (LJSpiller) [reply dynamics-gp/installation/installing-on-subsequent-computers.md:2]: Thanks @edupont04!

**Comment by LJSpiller** [comment]
@edupont04 - I don't have any further changes to this PR - I will be fixing other validation errors in another PR after this is merged because we have several PRs that need to be merged first. Can you review and merge this one? Thanks!

**Comment by LJSpiller** [comment]
Thanks @edupont04!

**Pull request merged**